client/state/plugins/installed/actions.js

```diff
@@ -450,19 +458,19 @@ function installPluginHelper(
 			return getPluginHandler( siteId, pluginData.slug ).install();
```

Comment:
Can we use `pluginId` here?

Reply:
@yashwin I'll test it again, but I remember this one in specific always requiring `pluginData.slug`